live-tests/README.adoc

```diff
@@ -51,11 +57,11 @@ To use this:
               $(which cargo-nextest) \
               root@YOUR_SCRIMLET_GZ_IP:/zone/oxz_switch/root/root
 
-3. On that system, unpack the tarball with:
+3. From the switch zone on that system, unpack the tarball with:
 
      tar xzf live-tests-archive.tgz
 
-4. On that system, run tests with:
+4. From the switch zone on that system, run tests with:
```

Comment:
I don't mind if we want to change this, but right now, this README is printing exactly what the tool prints out.  So we should change it there too if we're going to change it here.

Reply:
Oh I did not realize this was copy paste from that. I might like to change it there too then (I did figure it was supposed to run from the switch zone by looking at the file path, but it felt ambiguous to me)